soc/raspberrypi/rpi_pico/common/pinctrl_soc.h

```diff
@@ -14,23 +14,29 @@
  */
 struct rpi_pinctrl_soc_pin {
 	/** Pin number 0..29 */
-	uint32_t pin_num : 5;
+	uint32_t pin_num: 5;
 	/** Alternative function (UART, SPI, etc.) */
-	uint32_t alt_func : 5;
+	uint32_t alt_func: 5;
 	/** Maximum current used by a pin, in mA */
-	uint32_t drive_strength : 4;
+	uint32_t drive_strength: 4;
 	/** Slew rate, may be either false (slow) or true (fast) */
-	uint32_t slew_rate : 1;
+	uint32_t slew_rate: 1;
 	/** Enable the internal pull up resistor */
-	uint32_t pullup : 1;
+	uint32_t pullup: 1;
 	/** Enable the internal pull down resistor */
-	uint32_t pulldown : 1;
+	uint32_t pulldown: 1;
 	/** Enable the pin as an input */
-	uint32_t input_enable : 1;
+	uint32_t input_enable: 1;
 	/** Enable the internal schmitt trigger */
-	uint32_t schmitt_enable : 1;
+	uint32_t schmitt_enable: 1;
 	/** Output-enable override */
-	uint32_t oe_override : 2;
+	uint32_t oe_override: 2;
+	/** Output override */
+	uint32_t out_override: 2;
```

Comment:
Use a descriptive variable name, and then the comment becomes redundant, i.e.
```suggestion
	/** Output override */
	uint32_t output_override: 2;
```

Reply:
Since it matches the original naming that follows the SDK's gpio_set_oeover(), I think there is no problem with the current version.

Reply:
@soburi qnd @ajf58: Please let me know whether I should keep this naming or adapt with `input` and `output`, thanks!

Reply:
This is an area of ​​contention and a matter of style, so it's up to you to decide based on both sides' arguments.

Reply:
I would leave it as it is, so this is similar to the `oe-override config